src/index.js

```diff
@@ -1,4 +1,19 @@
-import './styles/main.sass';
-import laughing from './assets/laughing.svg';
-const laughImg = document.getElementById('laughImg');
-laughImg.src = laughing;
+import './styles/Sass/main.sass';
+import './assets/pokemon.png';
+import './assets/like.svg';
+import './assets/bikatshoo.svg';
+import './assets/1.svg';
+// import './assets/2.svg';
+// import './assets/3.svg';
+// import './assets/4.svg';
+// import './assets/5.svg';
+// import './assets/6.svg';
+// import './assets/7.svg';
+// import './assets/8.svg';
+// import './assets/9.svg';
```

Comment:
- [ ] Please remove any unused code from your codebase for a cleaner and more optimized environment. 🙏 see more tips [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.).

Reply:
Done